src/vma/dev/net_device_table_mgr.h

```diff
@@ -123,6 +123,7 @@ class net_device_table_mgr : public cache_table_mgr<ip_address, net_device_val*>
 	int 			    m_global_ring_pipe_fds[2];
 
 	uint32_t			m_max_mtu;
+	bool				m_ib_flow_spec_defined;
```

Comment:
Uninitialized member is causing coverity failure.
If not used - can be safely removed